invenio_drafts_resources/services/records/service.py

```diff
@@ -469,12 +470,12 @@ def rebuild_index(self, identity):
 
         return ret_val
 
-    def validate_draft(self, identity, id_):
+    def validate_draft(self, identity, id_, ignore_field_permissions=False):
```

Comment:
the flag is set to `False` by default to not break any existing workflows (although i doubt that this method was used anywhere outside of this module)